src/components/SignUp/Landing.tsx

```diff
@@ -252,4 +318,27 @@ const LoginLink = styled.p`
   }
 `;
 
+const CardHeading = styled.h3`
+  width: 225px;
+  top: 24%;
```

Comment:
I feel like without positioning, we shouldn't need top and bottom values here. The CardHeading text just fits at the top, and maybe we need to position it from the edges with extra margin or padding or something if removing these puts it too close to the edge (although I can't imagine that, since it doesn't have a position attribute to take them into account, like absolute or fixed).